packages/dx-scheduler-core/src/utils.js

```diff
@@ -90,6 +90,10 @@ export const findOverlappedAppointments = (sortedAppointments, byDay = false) =>
   return groups;
 };
 
+const isMidnight = date => date.isSame(date.clone().startOf('day'));
+const byDayConditional = (maxBoundary, startDate) => ((maxBoundary.isBefore(startDate, 'day'))
```

Comment:
`byDayConditional` looks a bit strange. Can we use something like `maxBoundaryPredicate`?

Reply:
Renamed